.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv:
+          - checkqa
+          - readme
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
```

You forgot to actually pass the arg:
```suggestion
        uses: actions/setup-python@v2
        with:
          python-version: ${{ matrix.python-version }}
```

---

I didn't pas it intentionally, because I wanted to use the default which is `3.x`.

---

You also used it in `name:`

---

I but it by a mistake, i'll remove it.

---

Actually I've reconsidered and put matrix.python-version = 3.x explicitly in 988dfbc. Thanks!